plugins/exporter.koplugin/clip.lua

```diff
@@ -392,10 +392,11 @@ function MyClipping:getDocMeta(view)
     if isEmpty(author) then
         author = isEmpty(parsed_author) and "Unknown Author" or parsed_author
     end
+    -- Remove illegal characters for the current OS' file names
+    local exportable_title = util.getSafeFilename(title)
```

Comment:
Doesn't really seem worth bothering with in this case?

Reply:
I'm a bit confused as I haven't contributed on github for long; since there's an approval is this comment towards me or mainly other members? I can "start a review" on my PR..? Sorry for the confusion 🙈

I think the name of the function is telling enough to stand on its own (if anything, maybe "exportable_title" isn't the best..? "output_filename" or something similar could be more expressive..? It's used just here and in another file to unpack the table, so it would be easy to change).

Reply:
Even if one `Approves` (it's just a polite flag for you and others, like "looks good to me"), this person and others can still comment and request changes.
What he meant is that your change could be a single line below, without the need for the local variable (and finding a nice name for it :):
`exportable_title =  util.getSafeFilename(title)`

And there, something jumps:
```diff
- exportable_title = parsed_title,
+ exportable_title =  util.getSafeFilename(title)
```
You're no more using `parsed_title`. Is that right?
Shouldn't it be:
```diff
- exportable_title = parsed_title,
+ exportable_title =  util.getSafeFilename(parsed_title)
```

Reply:
> I'm a bit confused as I haven't contributed on github for long; since there's an approval is this comment towards me or mainly other members? I can "start a review" on my PR..? Sorry for the confusion see_no_evil

All of the above. I meant it's fine with me as is, but I think it'd be be better to reduce this PR to a single line change instead of introducing a new short-lived variable that doesn't seem to help readability.

Reply:
@poire-z Thanks for the explanation on how the github part works :)
I did understand what the review meant! but I'm thankful for you taking the time to also explain that part 😄

`parsed_title` is "parsed" from the original file's name. 
Until now, that's what was used to name the output export file, instead of using the title from the file's metadata (which, if present, is probably more valuable/correct than the actual filename).
This was done as the title in the metadata could have illegal characters when it comes to filenames, which would crash the app.
In the steps before the part I've modified, there are conditional statements that turn "title" into the "parsed_title" already, so we can safely "sanitize" the title variable (which will be from the metadata, if present, or from the book's filename, if the metadata didn't have the title's value) and use that :)

Reply:
Alright, just wanted to be sure this was thought about and not an error :)

Reply:
Updated with inline function and renamed property (and all of its instances across the project, checked with grep from root)